.storybook/styles.css

```diff
@@ -33,6 +42,32 @@ body,
   max-width: 100px;
 }
 
+.isb-trigger {
+  display: inline-block;
+  width: var(--space-2);
+  height: var(--space-2);
+  border-radius: 50%;
+  background-color: var(--electric-blue-500);
```

Comment:
Maybe this doesn't apply to the storybook, but after reading the best practices, I think it said to use the theme variables and not the name of the actual color.

Reply:
yes in the front-end guide it says to use theme values. For the design system it's different, in each component we create local variables with their color value.
In the case of storybook, we use the variables directly, it's not a problem.